rasa/core/actions/forms.py

```diff
@@ -145,7 +143,9 @@ def _create_unique_entity_mappings(self, domain: Domain) -> Set[Text]:
         """
         unique_entity_slot_mappings = set()
         duplicate_entity_slot_mappings = set()
```

Comment:
Why do we still need to run all of the `FormAction`. Don't we need to comment out `activate` and the first line of `def do`?

Reply:
and wouldn't the `condition` part then go into `request_next_slot` / `is_done`?

Reply:
> Why do we still need to run all of the FormAction. Don't we need to comment out activate and the first line of def do?
Because we kept the concept of the `form`, I didn't want to remove anything in `FormAction`, only adapted bits and pieces here

> wouldn't the condition part then go into request_next_slot / is_done?
💡 I didn't consider these methods - those could be better ways for handling slot mapping condition.